### PR TITLE
Error handling page

### LINKED ISSUE #358: Error handling route
Currently error pages in production are simply blank pages

image

I think it would be helpful to have a general design for what we might want it to look like but as far as development it is very similar in construction to the 404 (`not-found.tsx`) page @sabakhilji just created.

perhaps a quick design mockup would be helpful though i think it could work if we kept it very similar to the design for the 404 page.

### PR DESCRIPTION
**Pull request by sabakhilji**
Closes #358

### CODE REVIEW

**Comment by benalleng** [comment]
Make sure to merge with master so that you are comparing with the most up to date code!

**Comment by sabakhilji** [comment]
> Make sure to merge with master so that you are comparing with the most up to date code!

So do I need to fetch master and then merge? Then again open a PR for this..

**Comment by benalleng** [comment]
Nope! just make sure to run git pull upstream master when you are working on new Pr's.  It looks like you went ahead and made the correct changes already!

**Review by benalleng — DISMISSED** [review]
Just make sure to pull from upstream and you should be G2G!

**Comment by benalleng** [comment]
I do see there is a conflict but you should be able to just accept your own changes

File: i18n/locales/en.ts [hunk 1]
```diff
@@ -373,12 +373,7 @@ const translations = {
     waiting: 'Waiting for your input above...',
   },
 
-  notfound: {
-    first: 'Hmm...where are we?',
-    second: 'It’s not quite clear where or what this place is.',
-    third: 'We might have gotten lost in space (or time).',
-    back_safety: 'Back to safety',
-  },
+
```
Comment (benalleng) [inline i18n/locales/en.ts:376]:
``` suggestion
notfound: {
    first: 'Hmm...where are we?',
    second: 'It’s not quite clear where or what this place is.',
    third: 'We might have gotten lost in space (or time).',
    back_safety: 'Back to safety',
  },
```

**Review by benalleng — APPROVED** [review]

**Pull request merged**